app/Models/CertificateModel.php

```diff
@@ -68,6 +69,32 @@ public function updateCertificate(string $certificateId, array $certificateData)
         }
     }
 
+    public function deleteCertificate(array $certificate)
+    {
+        try {
+            $this->db->transException(true)->transStart();
+            $isDeleted = $this->delete($certificate['certificateId']);
+            if (! $isDeleted) {
+                throw new Exception();
+            }
+
+            $fileModel = new FileModel();
+
+            $queryWhere = "uuid={$this->escape($certificate['previewUuid'])} OR uuid={$this->escape($certificate['certificateUuid'])}";
+            $isDeleted  = $fileModel->where($queryWhere)->delete();
```

Comment:
Instead of creating an or manually, you can make use of the whereIn statement
```suggestion
            $isDeleted  = $fileModel->whereIn('uuid', [$certificate['previewUuid'], $$certificate['certificateUuid']])->delete();
```